tls/s2n_x509_validator.c

```diff
@@ -277,20 +277,15 @@ s2n_cert_validation_code s2n_x509_validator_validate_cert_chain(struct s2n_x509_
                                                                 uint8_t *cert_chain_in, uint32_t cert_chain_len,
                                                                 s2n_pkey_type *pkey_type, struct s2n_pkey *public_key_out) {
 
-    if (!validator->skip_cert_validation && !s2n_x509_trust_store_has_certs(validator->trust_store)) {
-        return S2N_CERT_ERR_UNTRUSTED;
-    }
+    S2N_ERROR_IF(!validator->skip_cert_validation && !s2n_x509_trust_store_has_certs(validator->trust_store), S2N_CERT_ERR_UNTRUSTED);
```

Comment:
The return codes in the function like S2N_CERT_ERR_UNTRUSTED aren't actually s2n_errors: https://github.com/awslabs/s2n/blob/9802a051850e8bb8dbfe0f03d66fc9f6249d7bb3/tls/s2n_x509_validator.h#L30-L38

Reply:
ah will update. thanks